yrs/src/block.rs

```diff
@@ -452,7 +454,7 @@ pub(crate) struct Item {
 
     /// Used only when current item is used by map-like types. In such case this item works as a
     /// key-value entry of a map, and this field contains a key used by map.
-    pub parent_sub: Option<String>, //TODO: Rc since it's already used in Branch.map component
+    pub parent_sub: Option<Rc<str>>, //TODO: Rc since it's already used in Branch.map component
```

Comment:
Since `parent_sub` represents a static immutable string key, that's shared between branch map component and each of the blocks that are assigned as a value to that map's entry, it's much better to share them via `Rc` rather than deep copy the strings.